xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130b.xml

```diff
@@ -172,10 +178,32 @@ changes: Added @evidence to date
     <text>
         <body>
             <div type="edition">
-                <ab></ab></div>
+                <ab>
+                    <lb n="1"/>ὡ<unclear>ρ</unclear>χίζω ὑμᾶ<supplied reason="omitted">ς</supplied> διὰ τὼν
+                    <lb n="2"/>ἄνωμων καὶ δυσεβὴν
+                    <lb n="3"/><name key="Βαβύλας Babylas">Βαβυλᾶν</name> τὼν λαχανω
+                    <lb n="4" break="no"/>πώλιν: ὥσ<surplus>υ</surplus>πηρ ὑμῖς
```

Comment:
Colon or ano teleia?

Reply:
ano teleia I think; how should I encode that?